css/index.css

```diff
@@ -49,7 +50,61 @@
     font-weight: bolder;
     letter-spacing: 3mm;
 }
-
+body {
+    background-color: rgb(240, 221, 161);
+}
+#about {
+    text-align: center;
+    background-color: rgba(194, 137, 32, 0.918);
+    border: 3px dotted black;
+    margin: 10%
+}
+#experience {
+    text-align: center;
+    border: 3px dotted black;
+    margin: 5%;
+    background-color: rgb(194, 165, 69);
+}
+.headings {
+    text-align: center;
+    font-family: fantasy;
+    text-decoration: underline;
+}
+#skillsList {
+    border: 5px solid rgb(83, 58, 10);
+    margin-left: 40%;
+    margin-right: 40%;
+}
+#connect {
+    text-align: center;
+}
+.leavingMessages{
+    color:rgb(37, 26, 5)
+}
+.name {
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    color: rgb(233, 94, 2);
+}
+.email{
+    color: rgb(233, 94, 2);  
+}
+.messages{
+    color: rgb(233, 94, 2);
+}
+.submit{
+    background-color: rgb(170, 112, 36);
+    border: .5px none;
+    margin-left: 200%;
+}
+#footer{
```

Comment:
is there a special reason that you wrote the #footer personally designed with the border, text-align, font-family as this style?